torchok/models/necks/detection/fpn.py

```diff
@@ -60,3 +61,57 @@ class FPN(necks.FPN):
     def __init__(self, in_channels, **kwargs):
         in_channels = list(in_channels[::-1])
         super(FPN, self).__init__(in_channels=in_channels, **kwargs)
+
+    def forward(self, inputs):
```

Comment:
Where the diffrence with https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/necks/fpn.py ?

Reply:
Here was auto push to fp16 which I don't want to be always enabled